tests/integrationtest/t/executor/delete.test

```diff
@@ -135,6 +135,7 @@ create table parent2 (a int primary key);
 create table child2 (a int, foreign key (a) references parent2(a));
 insert into parent2 values (1), (2);
 insert into child2 values (1);
+-- replace_regex /`executor__delete`\.`child2`, CONSTRAINT `fk_1` FOREIGN KEY \(`a`\) REFERENCES `parent2` \(`a`\)/`executor__delete`.`child`, CONSTRAINT `fk_1` FOREIGN KEY (`a`) REFERENCES `parent` (`a`)/
```

Comment:
so we should move it to unit test and check error count > 0 and it's the expected one?

Reply:
Both are expected, and only (random) one of the error is reported. I think it'll have the same effect in unit test or integration test :thinking: .